client/lib/network-connection/test/index.js

```diff
@@ -1,32 +1,31 @@
-import config from '@automattic/calypso-config';
-import { expect } from 'chai';
-import sinon from 'sinon';
 import NetworkConnectionApp from 'calypso/lib/network-connection';
 
 describe( 'index', () => {
-	let configStub;
-
 	beforeEach( () => {
-		configStub = sinon.stub( config, 'isEnabled' ).returns( true );
-	} );
-
-	afterEach( () => {
-		configStub.restore();
+		jest.mock( '@automattic/calypso-config', () => {
```

Comment:
Isn't `jest.mock` usually called outside of any blocks and those mocks reset in e.g. `(before|after)Each` blocks? Not exactly sure if it makes a difference though.